cosmic_ray/worker.py

```diff
@@ -58,21 +62,44 @@ def worker(module_name,
     """
     try:
         module = importlib.import_module(module_name)
+        module_source_file = inspect.getsourcefile(module)
         module_ast = get_ast(module)
+        module_source = astunparse.unparse(module_ast)
+
         core = MutatingCore(occurrence)
         operator = operator_class(core)
-        operator.visit(module_ast)
+        # note: after this step module_ast and modified_ast
+        # appear to be the same
```

Comment:
I think yes. As far as I can tell the visit operation modifies the AST tree it operates on instead of returning another AST tree object. I've tried shuffling around some of the lines to prevent generating the diff if we don't have to and stumbled across this hence the comment.